lua-docs/content/reference/player.yml

```diff
@@ -39,6 +39,9 @@ properties:
     - name: "Velocity"
       type: "Number3"
       description: "Player's velocity (speed + direction)."
+     - name: "Wave"
+      type: "Boolean"
+      description: "Set the player animation wave to true or false."
```

Comment:
Be more precise about what the property does, like "when set to true, the Player starts waving, it stop when it is set to false". Also do not forget to put brackets around `Player` so it links to the correct page, you could also add ` characters around code words like true.